### PR TITLE
add more documentation about validation after building app with flatpak

### PR DESCRIPTION
**Pull request by pidddgy**
This would have helped me a lot and hopefully this will be useful to other developers.

### CODE REVIEW

File: flatpak-guide.md [hunk 1]
```diff
@@ -268,6 +268,10 @@ You already know the basics for:
     $ appstream-util validate /var/lib/flatpak/app/org.sugarlabs.ImplodeActivity/current/active/files/share/appdata/org.sugarlabs.ImplodeActivity.appdata.xml
     ```
 
+    If appstream-util validate is not running, you can also try `flatpak run org.freedesktop.appstream-glib validate` as documented in https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util
```
Comment (tchx84) [inline flatpak-guide.md:271]: Better to link the document than showing the whole URL. e.g. "as described in the [flathub documentation](https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util)."

File: flatpak-guide.md [hunk 2]
```diff
@@ -268,6 +268,10 @@ You already know the basics for:
...
+
+    If the validate command is having trouble reading the appdata XML file, you can try using `cat` to read the xml file, pipe the result into a local directory, and change the path that you pass into the validate function.
```
Comment (tchx84) [inline flatpak-guide.md:273]: Can you describe the scenario where this was necessary?

**Review by tchx84 — CHANGES_REQUESTED** [review]
Thanks for the clarifications, a few comments.

**Comment by pidddgy** [comment]
After thinking about it for a while, I think it would be best if we closed this PR - `flatpak run org.freedesktop.appstream-glib validate` was telling me things were okay when they weren't for some reason (hence why you had to return my task so many times, lol). I tried to use that because appstream-util wasn't installed and that alternative command was what I stumbled across first and I stupidly decided to use that instead. The workaround I had to do with `cat` and piping was also probably due to the alternative command as I just tested it with appstream-util  and it seems to be working fine.

 I think it would be better if we just included instructions on how to install appstream-util (it's a simple apt-get install)

**Pull request closed without merging**